packages/create-sitecore-jss/src/templates/nextjs-personalize/.env

```diff
@@ -0,0 +1,4 @@
+BOXEVER_CLIENT_KEY=
+BOXEVER_API=
+BOXEVER_TARGET_URL=
+GRAPH_QL_ENDPOINT=
```

Comment:
Let's assume the merge functionality is complete and remove the `GRAPH_QL_ENDPOINT` here since it's defined in the base .env.